core/common/src/main/java/alluxio/AbstractClient.java

```diff
@@ -173,6 +179,16 @@ public synchronized void connect() throws IOException, ConnectionFailedException
         afterConnect();
         checkVersion(getClient(), getServiceVersion());
         return;
+      } catch (IOException e) {
+        if (FRAME_SIZE_EXCEPTION_PATTERN.matcher(e.getMessage()).find()) {
+          // See an error like "Frame size (67108864) larger than max length (16777216)!",
+          // pointing to the helper page.
+          LOG.error("Failed to set up a connection to " + getServiceName() + " "
+              + mMode + " @ " + mAddress + " due to the mismatched transport. Please refer to "
+              + RuntimeConstants.ALLUXIO_DOCS_URL + "/en/Debugging-Guide.html "
+              + "for the possible reasons and suggestions.");
+        }
+        throw e;
```

Comment:
instead of both logging and throwing exception, I wrap and throw a new exception with the referenced page in message.